0.990 reports error for function with body that is just ellipsis

mypy 0.990 (python 3.8) reports an error for the following source code
```py
def foo() -> int:
    ...
```
```
$ mypy foo.py 
/tmp/foo.py:1: error: Missing return statement  [empty-body]
Found 1 error in 1 file (checked 1 source file)
```
mypy 0.981 would let the same code pass. Was that an intended change in between, or a regression?

From [mypy CLI documentation](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-warn-no-returnl)
> By default, mypy will generate errors when a function is missing return statements in some execution paths. The only exceptions are when:
> - The function has a None or Any return type
> - The function has an empty body or a body that is just ellipsis (...). Empty functions are often used for abstract methods.

I'd expect Success for the given example, because the 2nd exception ("body that is just ellipsis") applies.

--- a/docs/source/command_line.rst
+++ b/docs/source/command_line.rst
@@ -448,9 +448,10 @@ potentially problematic or redundant in some way.
     are when:
 
     -   The function has a ``None`` or ``Any`` return type
-    -   The function has an empty body or a body that is just
-        ellipsis (``...``). Empty functions are often used for
-        abstract methods.
+    -   The function has an empty body and is marked as an abstractmethod,
+        is in a protocol class, or is in a stub file
+    -   For unreachable execution paths, for example, if an exception
+        is always raised
 
     Passing in :option:`--no-warn-no-return` will disable these error
     messages in all cases.

--- a/docs/source/command_line.rst
+++ b/docs/source/command_line.rst
@@ -450,7 +450,7 @@ potentially problematic or redundant in some way.
     -   The function has a ``None`` or ``Any`` return type
     -   The function has an empty body and is marked as an abstract method,
         is in a protocol class, or is in a stub file
-    -  The execution path is unreachable; for example, if an exception
+    -  The execution path can never return; for example, if an exception
         is always raised
 
     Passing in :option:`--no-warn-no-return` will disable these error

--- a/docs/source/command_line.rst
+++ b/docs/source/command_line.rst
@@ -448,9 +448,9 @@ potentially problematic or redundant in some way.
     are when:
 
     -   The function has a ``None`` or ``Any`` return type
-    -   The function has an empty body and is marked as an abstractmethod,
+    -   The function has an empty body and is marked as an abstract method,
         is in a protocol class, or is in a stub file
-    -   For unreachable execution paths, for example, if an exception
+    -  The execution path is unreachable; for example, if an exception
         is always raised
 
     Passing in :option:`--no-warn-no-return` will disable these error

